rpcs3/Emu/Cell/lv2/sys_usbd.cpp

```diff
@@ -132,6 +145,8 @@ class usb_handler_thread
 	shared_mutex mutex_sq;
 	ppu_thread* sq{};
 
+	u64 usb_hotplug_timeout = 0;
```

Comment:
With your recent changes, this is accessed outside the usb thread, please make it `atomic_t<u64>`.